### PR TITLE
AKU-551: Retain item selection when switching views

### PR DESCRIPTION
**Pull request by draperd**
This PR addresses https://issues.alfresco.com/jira/browse/AKU-551 to ensure that when switching views any item selection on one view is retained on the next.

### CODE REVIEW

File: aikau/src/main/resources/alfresco/menus/AlfMenuBarSelectItems.js [hunk 1]
```diff
@@ -176,19 +176,22 @@ define(["dojo/_base/declare",
        * @param {object} payload
        */
       determineSelection: function alfresco_menus_AlfMenuBarSelectItems__determineSelection(payload) {
-         if (payload.availableItemCount != null && payload.selectedItemCount != null) 
+         var available = parseInt(payload.availableItemCount, 10);
```
Comment (tunaaksoy) [inline aikau/src/main/resources/alfresco/menus/AlfMenuBarSelectItems.js]: if payload.availableItemCount or payload.selectedItemCount will be null or undefined NaN will be returned.

Comment (draperd) [inline aikau/src/main/resources/alfresco/menus/AlfMenuBarSelectItems.js]: ...but then they'll just fail the next if block? Which I think will be fine?

Comment (tunaaksoy) [inline aikau/src/main/resources/alfresco/menus/AlfMenuBarSelectItems.js]: Ok, that's true. Would it make sense to log this then in an else statement?

Comment (draperd) [inline aikau/src/main/resources/alfresco/menus/AlfMenuBarSelectItems.js]: Yes, I can make that change

**Pull request merged**